.vib/opencart/cypress/cypress/integration/opencart_spec.js

```diff
@@ -66,46 +49,9 @@ it('allows an admin to add a product to the catalog', () => {
     cy.get('[data-original-title="Save"]').click();
     cy.contains('Success');
     cy.visit('/');
-    cy.get('.form-control').type(
+    cy.get('[name="search"]').type(
       `${product.newProduct.name}.${random} {enter}`
     );
     cy.get('#content').contains(`${product.newProduct.name}.${random}`);
   });
 });
-
-it('checks if SMTP is configured as per the chart', () => {
-  cy.login();
-  cy.contains('System').click();
-  cy.contains('Settings').click();
-  cy.get('[data-original-title="Edit"]').click();
-  cy.contains('[data-toggle="tab"]', 'Mail').click();
-  cy.get('#input-mail-smtp-hostname').should(
-    'have.value',
-    Cypress.env('smtpHost')
-  );
-  cy.get('#input-mail-smtp-username').should(
-    'have.value',
-    Cypress.env('smtpUser')
-  );
-});
-
-it('allows the admin to upload a file', () => {
```

Comment:
Would it be possible to have the image addition test included in the one testing a product addition? Usually, the image test is useful to check the asset's PVC is working properly.

Reply:
I decided to remove the image upload test because it does not add any value. The app is configured by default to only persist the following files: `config.php` and `admin/config.php`.

https://github.com/bitnami/containers/blob/3694619b3152976363111b80487e93e3fd9214bd/bitnami/opencart/3/debian-11/rootfs/opt/bitnami/scripts/opencart-env.sh#L74

I manually performed the test case and checked that the image is not stored in the PVC, so I don't think this is useful here.